### DISCUSSION TITLE
Cannot install the TRF model from the Hugging Face Hub if the CPU model is installed

### DISCUSSION THREAD

**Discussion by DavidNemeskey** [huspacy/huspacy#52] [Q&A] [answered]
**Describe the bug**
If I try to install the TRF model from the HFH, I get the following error:

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
hu-core-news-lg 3.5.0 requires spacy<3.6.0,>=3.5.0, but you have spacy 3.4.4 which is incompatible.
```

Ideally, *all* models have the same dependencies so that we can install both CPU and GPU variants in the same environment.

**Comment by oroszgy** [comment, marked as answer]
Actually you can, just make sure that each model depends on the right spaCy version. The problem in your case is that you are trying to install `hu_core_news_lg` v3.5.0 which depends on `spacy>=3.5,<3.6`, while you already have `hu_core_news_lg` v3.4.0 on your environment which requires `spacy>=3.4,<3.5`. 

The solution would be to [install](https://huspacy.github.io/reference/huspacy/__init__/) an [older large model](https://huspacy.github.io/models/changelog_lg/), which has the same spacy dependcy as the transfomer model.

A few additional notes:
- Each model version is tagged in HF Hub, you can [install them directly using pip](https://huspacy.github.io/models_gen/install_lg/): ![Screenshot_20230208_131700](https://user-images.githubusercontent.com/145334/217527549-277f02df-3e0b-4ee4-b7ac-beb0beffbb8e.png)
- HuSpaCy's model versions follows [spaCy's versioning scheme](https://spacy.io/models#model-versioning), this means that 3.x.y HuSpaCy models are compatible with spacy 3.x only.
- There are no transformer models available for spaCy 3.5, as we build on [epxerimental features](https://github.com/explosion/spacy-experimental) which are not yet compatible with the latest release.
